core/db_models/EEM_Datetime.model.php

```diff
@@ -150,40 +154,97 @@ protected function __construct($timezone)
      * @access public
      * @return EE_Datetime[] array on success, FALSE on fail
      * @throws EE_Error
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws ReflectionException
+     * @throws InvalidInterfaceException
      */
     public function create_new_blank_datetime()
     {
         // makes sure timezone is always set.
         $timezone_string = $this->get_timezone();
+        $start_date = apply_filters(
+            'FHEE__EEM_Datetime__create_new_blank_datetime__start_date',
+            $this->current_time_for_query('DTT_EVT_start', true) + MONTH_IN_SECONDS
+        );
+        $end_date = apply_filters(
+            'FHEE__EEM_Datetime__create_new_blank_datetime__end_date',
+            $this->current_time_for_query('DTT_EVT_end', true) + MONTH_IN_SECONDS
+        );
         $blank_datetime  = EE_Datetime::new_instance(
             array(
-                'DTT_EVT_start' => $this->current_time_for_query('DTT_EVT_start', true) + MONTH_IN_SECONDS,
-                'DTT_EVT_end'   => $this->current_time_for_query('DTT_EVT_end', true) + MONTH_IN_SECONDS,
+                'DTT_EVT_start' => $start_date,
+                'DTT_EVT_end'   => $end_date,
                 'DTT_order'     => 1,
                 'DTT_reg_limit' => EE_INF,
             ),
             $timezone_string
         );
+        $start_time = apply_filters(
```

Comment:
It would probably be good to add a comment here defining the format of the value being filtered (eg it took me a while to figure out what the "ga" meant).